scripts/platforms/workday_apply.py

```
@@ -0,0 +1,394 @@
+                ctx = await extract_field_context_generic(ta)
+                question = ctx["combined"]
+                if not question:
+                    continue
+                ck = f"wk::ta::{normalize_key(question)}"
+                if ck not in generated_cache:
+                    generated_cache[ck], _ = generate_structured_answer(question=question, answer_kind="long_text")
+                ans = generated_cache[ck]
+                if ans:
+                    await ta.fill(ans)
+                    field_memory[normalize_key(question)] = ans
+                    print(f"Textarea filled")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)